script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -462,27 +462,27 @@ It's as though he were grinning in agreement with Arcueid's provocation.
 [sha:b8aaf187460b84ead817cb5a6b19a3c96c926449]{
 -- Page 548, Offset 10071.
 -- 「最も強きものとは千変にして<偏|かたよ>らぬもの。
-"The strongest ones are those who continue to adapt and do not remain locked in their ways.
+"Those that strong be... retain unfaltering through vicissitudes do.
```

Comment:
idk if it's just early or if this line is utter nonsense

Reply:
Maybe "remain" instead of "retain", actually.